2019/LPC/README.md

```diff
@@ -0,0 +1,24 @@
+## Conference: Linux Plumbers Conference
```

Comment:
@artur-rs there should be link to etherpad there are very important notes:
https://etherpad.net/p/LPC2019_System_Boot_and_Security